joern-cli/frontends/c2cpg/src/test/scala/io/joern/c2cpg/io/JSONCompilationDatabaseParserTests.scala

```diff
@@ -111,10 +111,10 @@ class JSONCompilationDatabaseParserTests extends AnyWordSpec with Matchers {
           |    "file": "file2.cc" }
           |]""".stripMargin
 
-      BetterFile.usingTemporaryFile("compile_commands.json") { commandJsonFile =>
-        commandJsonFile.writeText(content)
+      FileUtil.usingTemporaryFile("compile_commands.json") { commandJsonFile =>
+        Files.writeString(commandJsonFile, content, Charset.defaultCharset())
```

Comment:
Please create a `write` or `writeString` named function in FileUtil to wrap `Files.writeString`. The goal is to not always require people to give `Charset.defaultCharset()` as argument.